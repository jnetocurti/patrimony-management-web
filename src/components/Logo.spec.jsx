import React from 'react';

import { shallow } from 'enzyme';

import Logo from './Logo';

describe('src/components/Logo', () => {
  it('should render correctly', () => {
    const component = shallow(<Logo />);
    expect(component).toMatchSnapshot();
  });
});
