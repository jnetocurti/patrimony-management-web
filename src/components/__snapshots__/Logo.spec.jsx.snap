// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Logo should render correctly 1`] = `
<img
  alt="Logo"
  src="/static/logo.png"
/>
`;
